<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QMainWindow" name="View">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>View</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>140</y>
      <width>981</width>
      <height>711</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="selectRegionPage">
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>801</width>
        <height>691</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QPushButton" name="romeButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white;
padding: 25px;
border:5px  solid black;
border-radius: 100px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="englandButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white;
padding: 25px;
border:5px  solid black;
border-radius: 100px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="greeceButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white;
padding: 25px;
border:5px  solid black;
border-radius: 100px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="franceButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white;
padding: 25px;
border:5px  solid black;
border-radius: 100px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="TriviaPage">
     <widget class="QWidget" name="verticalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>801</width>
        <height>241</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="regionLayout_2">
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLabel" name="questionLabel">
         <property name="font">
          <font>
           <family>Papyrus</family>
           <pointsize>21</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">padding: 15px;
background-color: #7C0A02;
color:white;</string>
         </property>
         <property name="text">
          <string>Insert the text for each trivia question here</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>310</y>
        <width>801</width>
        <height>315</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="answerOne">
         <property name="font">
          <font>
           <family>Papyrus</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">padding: 15px;
background-color: black;
color:white;</string>
         </property>
         <property name="text">
          <string>Answer 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="answerTwo">
         <property name="font">
          <font>
           <family>Papyrus</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">padding: 15px;
background-color: black;
color:white;</string>
         </property>
         <property name="text">
          <string>Answer 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="answerThree">
         <property name="font">
          <font>
           <family>Papyrus</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">padding: 15px;
background-color: black;
color:white;</string>
         </property>
         <property name="text">
          <string>Answer 3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="answerFour">
         <property name="font">
          <font>
           <family>Papyrus</family>
           <pointsize>16</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">padding: 15px;
background-color: black;
color:white;</string>
         </property>
         <property name="text">
          <string>Answer 4</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="backButton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>640</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Papyrus</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(47, 47, 47);
color:white;</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nextButton_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>640</y>
        <width>571</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Papyrus</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(47, 47, 47);
color:white;</string>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="gamePage">
     <widget class="QLabel" name="placeholder">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>681</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Papyrus</family>
        <pointsize>28</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 15px;
background-color: #7C0A02;
color:white;</string>
      </property>
      <property name="text">
       <string>Go back to trivia</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="goBackBtn">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>180</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:black;
color:white;</string>
      </property>
      <property name="text">
       <string>Go Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>520</y>
        <width>461</width>
        <height>95</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="triviaBtn">
         <property name="styleSheet">
          <string notr="true">background-color:white;
color:black;</string>
         </property>
         <property name="text">
          <string>Trivia</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="imageBtn">
         <property name="styleSheet">
          <string notr="true">background-color:white;
color:black;</string>
         </property>
         <property name="text">
          <string>Images</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="goBackBtn_2">
         <property name="styleSheet">
          <string notr="true">background-color:white;
color:black;</string>
         </property>
         <property name="text">
          <string>Go Back</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="infoLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>831</width>
      <height>711</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Papyrus</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 25px;
background-color: #7C0A02;
color:white;</string>
    </property>
    <property name="text">
     <string>Info</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="regionLabel">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>20</y>
      <width>831</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Papyrus</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 15px;
background-color: #7C0A02;
color:white;</string>
    </property>
    <property name="text">
     <string>Region</string>
    </property>
   </widget>
   <widget class="QPushButton" name="freePlayBtn">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>870</y>
      <width>801</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(202, 58, 58);
color:white;</string>
    </property>
    <property name="text">
     <string>Free Play Mode</string>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>20</y>
      <width>801</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Papyrus</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">padding: 15px;
background-color: #7C0A02;
color:white;</string>
    </property>
    <property name="text">
     <string>Score: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="PlayNarBtn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>870</y>
      <width>451</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(202, 58, 58);
color:white;</string>
    </property>
    <property name="text">
     <string>Play Narration</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextBtn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>870</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(202, 58, 58);
color:white;</string>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="previousBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>870</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(202, 58, 58);
color:white;</string>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:gray;</string>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionExit"/>
    <addaction name="actionHelp"/>
    <addaction name="actionSources"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_Aa">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionSources">
   <property name="text">
    <string>Sources</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
